README.md

```diff
@@ -11,17 +11,14 @@ on:
   push:
     branches:
       - master
-name: Deploy master branch
+name: Serverless deploy master branch
 jobs:
   deploy:
     name: deploy
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@master
-    - name: npm install
-      uses: actions/npm@master
-      with:
-        args: install
+    - run: npm ci
```

Comment:
I believe that in order for `npm ci` to work you first need to install npm since it is not available by default in the github workflow virtual environment. for that you could use this action: https://github.com/marketplace/actions/setup-node-js-for-use-with-actions

Reply:
From what I can tell, `npm ci` is meant to be used in automated environments such as test platforms, continuous integration, and deployment – or any situation where you want to make sure you’re doing a clean install of your dependencies. [source](https://docs.npmjs.com/cli/ci.html)

I am currently using an action that runs `npm ci`. It appears to work fine for me.

Reply:
I think you're misunderstanding my comment.
the use of `npm ci` is fine, but you will first need to install node+npm as part of the workflow.